plugins/amazonq/shared/jetbrains-community/src/software/aws/toolkits/jetbrains/services/cwc/controller/chat/telemetry/QTelemetryUtils.kt

```diff
@@ -12,3 +12,8 @@ fun getStartUrl(project: Project): String? {
     val connection = ToolkitConnectionManager.getInstance(project).activeConnectionForFeature(QConnection.getInstance()) as? AwsBearerTokenConnection?
     return connection?.startUrl
 }
+
+fun getIsAmazonStartUrl(project: Project): Boolean {
+    val startUrl = getStartUrl(project)
+    return startUrl != null && startUrl == "https://amzn.awsapps.com/start"
+}
```

Comment:
i dont think we should hardcode this in the public toolkit. the internal builder team already sets up values for internal users and project context should just be another value defined by them